scripts/dependencies/common_tools.sh

```diff
@@ -34,36 +34,37 @@ function waitForSuccess() {
     done
 }
 
-# Installs an operator according to given subscription resource and waits, params: operatorSubscriptionResource
+# Installs an operator according to given subscription resource and waits, params: operatorSubscriptionResource, namespace (defaults to openshift-operators)
 # Example:
 #  install_operator_and_wait serverless/serverlessSub.yaml
 #
 function install_operator_and_wait()
 {
   local resource=$1
+  local namespace=${2:-openshift-operators}
```

Comment:
I don't like magic defaulting in general, and I don't like it even more in shell scripts. I personally prefer not do any defaulting and gracefully failing if argument is missing, but that means changing everywhere where this function is used.

Let me know what your preference is and I can change.